runtime/src/main/resources/application.yml

```diff
@@ -80,3 +80,11 @@ generator:
 controllers:
   integration:
     enabled: true
+
+client:
+  state:
+    authenticationAlgorithm: HmacSHA1
```

Comment:
Yeah, that's what I was planing on doing, but in a separate PR as this needs some thinking behind it. Right now you can override these via environment variables. Not sure if we can have them dynamically set in the OpenShift templates.

Things to think about:
 - do we need this statically or dynamically (think regenerate keys every n days)
 - do we do this via k8s secrets
 - if dynamically how to handle the transition (need n-1 keys)
 - if dynamically then we need key distribution (spring cloud k8s, but it reloads the app context)